resources/webroot/index.js.mustache

```diff
@@ -2664,35 +2664,35 @@ function getPokestopPopupContent (pokestop) {
   content += '<div class="text-left">'
   if (isPoweredUp) {
     content += 
-      '<b>Power Up Level:</b> ' + pokestop.power_up_level + '<br>' +
-      '<b>Power Up End:</b> ' + powerUpEndDate.toLocaleTimeString() + ' (' + getTimeUntill(powerUpEndDate) + ')<br><br>'
+      '<b>'+i18n('map_gym_power_up_level')+'</b> ' + pokestop.power_up_level + '<br>' +
```

Comment:
This could be also used for pokestop, please rename the key do it matches both :)

Reply:
Fixed!